### ISSUE TITLE
can you please specify individual nltk packages.

### ISSUE THREAD

**Issue by manishchandwani** [Exaphis/HackLoco-Trivia#3]
the whole database is in Gb's . Also after putting bearer and device id the bot only shows the time of quiz show but when quiz starts i got this 
[Not in json]
and after that it again started showing the quiz time whereas the quiz already started.

**Comment by surendraraju** [comment]
Yes, after setting up everything it does nothing but showing the next game time.
